src/drivers/driver_common/driver_common.c

```diff
@@ -88,7 +88,8 @@ void _starpu_driver_start_job(struct _starpu_worker *worker, struct _starpu_job
 		task->status = STARPU_TASK_RUNNING;
 
 		STARPU_AYU_RUNTASK(j->job_id);
-		cl->per_worker_stats[workerid]++;
+		if (profiling && profiling_info)
```

Comment:
checking for (the per-task) `profiling_info` is not needed here since it's about the codelet